src/main/resources/db/migration/V1_6__alter_svarfrist_type_to_date.sql

```diff
@@ -0,0 +1,19 @@
+ALTER TABLE varsel
+    ADD COLUMN svarfrist_temp DATE;
+
+UPDATE varsel
+SET svarfrist_temp = DATE(svarfrist)
+WHERE true;
+
+ALTER TABLE varsel
+    DROP COLUMN svarfrist;
+
+ALTER TABLE varsel
+    ADD COLUMN svarfrist DATE;
+
+UPDATE varsel
+SET svarfrist = DATE(svarfrist_temp)
+WHERE true;
+
+ALTER TABLE varsel
+    DROP COLUMN svarfrist_temp;
```

Comment:
Legge til dette også?

ALTER TABLE varsel
    ALTER COLUMN svarfrist SET NOT NULL;

Reply:
Ja! Bra sett 🙌